### PR TITLE
v1.0.5

### PR DESCRIPTION
**Pull request by boxdrop**
Changed logic to always use displayAdminOrder

### CODE REVIEW

File: upgrade/Upgrade-1.0.5.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * Upgrades a module to 1.0.5.
+ * Removes old hook
+ * 
+ * @author sweber <sw@boxdrop.com>
+ * @param  mixed  $object
+ * @return boolean
+ */
+function upgrade_module_1_0_5($object)
+{
+	return ($object->registerHook('displayAdminOrderContentShip'));
```
Comment (Quetzacoalt91) [inline upgrade/Upgrade-1.0.5.php]: I do not understand how this update can be useful. You removed this hook in boxdropshipment.php, and it is also not implemented anymore.

**Comment by boxdrop** [comment]
Copy and paste error. That hook should be unregistered. Basically we moved from displayAdminOrderContentShip to displayAdminOrder. Pushed a fix.

**Comment by boxdrop** [comment]
thanks

**Comment by Quetzacoalt91** [comment]
Hi @boxdrop,

Thanks, your PR has just been merged. The new version should be available from tomorrow.

Regards

**Pull request merged**